app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,72 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all.order(:created_at)
+  end
+
+  def show
+    task_id = params[:id].to_i
```

Comment:
Actually, you don't need to do `.to_i`, and you can just do `task_id = params[:id]`! That's because Rails lets us get away with this.